### PR TITLE
TT-331 redesign of the sidebar

### PR DESCRIPTION
**Pull request by edgardavid2015**
# Problem
The current sidebar of the time tracker application is OK, it's working and has no functionality errors. But this sidebar could have a better design that enhances the user experience.

current_sidebar

# Solution
In this pull request, the sidebar has a modern design, with new icons, new colors, and new functionality.

new_sidebar

### CODE REVIEW

**Review by scastillo-jp — APPROVED** [review]

**Review by JosueOb — CHANGES_REQUESTED** [review]
Hi Edgar, I have checked your changes and everything works fine but I see that you are not using the color palette set for the project and you are not considering the changes for the dark mode.

File: src/app/modules/shared/components/sidebar/sidebar.component.scss [hunk 1]
```diff
@@ -65,7 +79,7 @@ body {
 }
 
 .dark-mode-toggle {
```
Comment (JosueOb) [inline src/app/modules/shared/components/sidebar/sidebar.component.scss:81]: This style is not used, it would be better to remove it.

**Review by JosueOb — COMMENTED** [review]
I have finished reviewing the changes made and I have some suggestions, the first one is about the icons, I see that some SVG icons are still kept in the assets directory, I think those icons that are not used should be removed, and the position of the icon to show or hide the sidebar does not keep its initial position.

**Review by JosueOb — APPROVED** [review]

File: src/app/modules/shared/components/sidebar/sidebar.component.scss (hunk 1 above)
Comment (edgardavid2015) [reply src/app/modules/shared/components/sidebar/sidebar.component.scss:81]: Thanks for your review Josue, I will delete it in the next commit.

**Pull request merged**